website/docs/components/event_bus/event_bus_plugin.md

```
@@ -0,0 +1,362 @@
+});
+
+// All operations through `eventBus` will use "app:..." event names
+await eventBus.addListener("user.created", listener);
+await eventBus.dispatch("user.created", data);
+```
+
+### Before/after behavior
+
+**Before** — Event names are used as-is:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)